### DISCUSSION TITLE
Schönes Projekt, funktioniert und macht Spass zum Nachbauen (3D-Gehäuse?)

### DISCUSSION THREAD

**Discussion by KuhnDaniel** [beabel/radiowecker#14] [Fragen & Antworten / Q&A] [answered]
Hallo in die Runde...
seit nun 1. Woche werkelt das ganze mit deinem Radiowecker Script/Fork!
Danke für das Überarbeiten und das Bereitstellen.

Ich mache mir gerade Gedanken mit einem netten Gehäuse (3D-Druck). 
Das Gehäuse mit dem Frontdeckel vom Display AZ-Touch zu integrieren ist zwar funktionell - zur Halterung ok - aber da geht sicher noch etwas.  Ein Austausch wäre schön. 

Freu mich auf... was da kommt!
Digitale Grüsse Daniel

**Comment by beabel** [comment]
Was gefällt dir nicht an dem was im Blog als Druckvorlage zur Verfügung gestellt wird?
[3D STL Datei](https://github.com/beabel/radiowecker/files/13313514/AZ_Touch_Pult.zip)

**Comment by KuhnDaniel** [comment]
Es wirkt halt aufgesetzt, was es ja auch ist. Auch zum Drucken ist es etwas aufwendig mit Überhang.
Aber klar, die Schräge verlangt nach etwas Unterstützung oder wie hast du das gedruckt?

**Reply by beabel** [reply]
Ich bin ehrlich, ich habe keinen 3d Drucker, ich habe drucken lassen.
Und so ein Gehäuse in 3D zu entwerfen könnte ich sicherlich auch nicht, da noch nie versucht.

**Comment by paulchen67** [comment]
Hallo, mir hat das vorhandene Gehäuse auch nicht sonderlich zugesagt. Da ich einen 3D Drucker habe und dafür sehr viel selber konstruiere und auch das nötige CAD Programm dafür vorhanden ist, habe ich ein neues Gehäuse konstruiert. Bei Interesse kann ich das gerne teilen.

![image](https://github.com/beabel/radiowecker/assets/80068451/97caba3d-b90f-4bcc-9c22-7266258e6670)
![image](https://github.com/beabel/radiowecker/assets/80068451/89fbbc4f-cb87-41a6-82fd-c35253c40b7e)
![image](https://github.com/beabel/radiowecker/assets/80068451/48a3eeb7-7359-4aca-9ef3-f9df31d26e7a)

Da ich von vornherein denn Radiowecker so haben wollte, dass ich meine Apple Watch damit über Nacht auch gleich laden wollte, sieht das Ganze bei mir dann so aus.

![image](https://github.com/beabel/radiowecker/assets/80068451/08d0a3d6-dcf3-42ef-aa0f-41d4aca83493)
![image](https://github.com/beabel/radiowecker/assets/80068451/2780cd41-000c-4f4d-87da-5765f17e49a6)

**Reply by beabel** [reply]
Das sieht ja mal sehr schick aus.

Was sind das für kleine "Pömpel" an der Rückseite?

**Reply by KuhnDaniel** [reply]
Das ist doch schon richtig schick und funktional...wow
Wie hast du da das Display/Grundplatte befestigt auf der Innenseite?
Ich habe auch selber 3D Drucker und Fusion360, da ich immer wieder selber Gehäuse benötige für IoT Projekte mit Schüler:Innen.

Besten Dank und ja mein Interesse am stl ohne Zusatz/Ladestation wäre gegeben.
Beste digitale Grüsse
Daniel

**Reply by paulchen67** [reply]
> Das sieht ja mal sehr schick aus.
> 
> 
> 
> Was sind das für kleine "Pömpel" an der Rückseite?

Das sind die Stecker für die JST- Buchsen.
Hatte ich vergessen vor dem Screenshot auszublenden.
Ich habe einige oft benötigte Elektronik Bauteile gezeichnet und abgespeichert. Dann muss ich sie beim Bedarf nur reinkopieren und an den richtigen Platz schieben. Dient auch zur Kontrolle ob ich die Aufnahme dafür richtig gezeichnet habe.

Im Radiowecker Chat von AZ Delivery bin ich Dennis.
Vielen Dank für deine Antworten, super Projekt.

**Reply by paulchen67** [reply]
> Das ist doch schon richtig schick und funktional...wow
> 
> Wie hast du da das Display/Grundplatte befestigt auf der Innenseite?
> 
> Ich habe auch selber 3D Drucker und Fusion360, da ich immer wieder selber Gehäuse benötige für IoT Projekte mit Schüler:Innen.
> 
> 
> 
> Besten Dank und ja mein Interesse am stl ohne Zusatz/Ladestation wäre gegeben.
> 
> Beste digitale Grüsse
> 
> Daniel
> 
> 

Das Display wird durch die Vorder- und Rückseite in Position gehalten. Die 2 Hälften werden durch die 8 M3x25
VA Inbus Linsenkopfschrauben zusammengezogen und klemmen dadurch das Display.

![image](https://github.com/beabel/radiowecker/assets/80068451/2139e8c9-b332-4512-8461-fbe44a24e3e3)

In die gedruckten 8 Bohrungen müssen mit einem M3 Gewinndeschneider nur noch die Gewinde geschnitten werden. Die 2 Gehäusehälften können ohne Stützen gedruckt werden.

Kann ich hier auch Dateien hochladen?
Dann könnte ich die STL Dateien für den Drucker hier reinstellen.

Ich konstruiere mit Shapr3D am iPad.

**Reply by beabel** [reply]
> Kann ich hier auch Dateien hochladen?
Dann könnte ich die STL Dateien für den Drucker hier reinstellen.

zip Dateien sollten funktionieren.
ansonsten habe ich im Profil jetzt auch eine Mail Adresse hinterlegt.

Das wäre Klasse, dann kann ich dem Projekt auch noch einen Ordner STL hinzufügen.

**Reply by paulchen67** [reply]
> > Kann ich hier auch Dateien hochladen?
> 
> Dann könnte ich die STL Dateien für den Drucker hier reinstellen.
> 
> 
> 
> zip Dateien sollten funktionieren.
> 
> ansonsten habe ich im Profil jetzt auch eine Mail Adresse hinterlegt.
> 
> 
> 
> Das wäre Klasse, dann kann ich dem Projekt auch noch einen Ordner STL hinzufügen.

ZIP Datei konnte ich keine hochladen, werde es über die Mailadresse versuchen.

**Comment by paulchen67** [comment, marked as answer]
Interessant, in der GitHub App fürs Tablet gibts im Gegensatz zur Browserversion die Büroklammer zum hochladen von Dateien nicht.

Hier die STL Dateinen für den 3D Drucker:
[Exportdateinen.zip](https://github.com/beabel/radiowecker/files/13325415/Exportdateinen.zip)

Inhalt
6 STL-Dateien
1 TXT- Datei mit der Stückliste der notwendigen Schrauben

**Reply by KuhnDaniel** [reply]
Besten Dank für deine Arbeit und die Freigabe zur Nutzung! Lege mir das mal auf meinen Kobra2 Max :-)
Beste digitale Grüsse
Daniel

**Reply by KuhnDaniel** [reply]
Habe das ganze nun gedruckt.. perfekt, sitzt alles wie angegossen, passt alles wie es soll!
Mit 3 Std 20Min das ganze Gehäuse gedruckt. Vielen herzlichen Dank.
Gruss Daniel

**Comment by paulchen67** [comment]
Freut mich das alles passt. Nur die Lautsprecher gehen etwas stramm rein, ist aber noch im grünen Bereich.

Ich mache vielleicht noch eine schmälere Version für die Variante (Cheap) wo keine AZ-Touch Mod vorhanden ist und das Display direkt mit dem ESP 32 verbunden wird. Dadurch kann das Ganze schmäler werden weil die doch sehr breite Platine vom AZ-Touch wegfällt.

**Reply by DarnokHillstar** [reply]
Hi paulchen67,

gibt es die schmälere Variante schon? Könnstest Du die STL Dateien wieder zur Verfügung stellen?
Danke

**Reply by paulchen67** [reply]
Nein,die gibt es nicht. Ist auch aktuell nicht geplant. Sorry.

**Reply by DarnokHillstar** [reply]
Sorry, - ich nochmal: 

Die Datei "Auflage Verstärker" ist mir gerade nicht so klar. Wo wird die, für was befestigt? Kannst Du da noch ein Foto hier reinlegen? Danke

Drucke gerade. Vorderseite hat 3,5 Stunden gedauert. Gehäuse Rückseite läuft gerade. Wird bei über 8 Stunden rauskommen.
Also, so schnell, wie oben gesagt, wird das nichts (Anycube i3 Mega).

Aber das was bisher rausgekommen ist, - ist toll. Danke. Wird ein Geschenk für meine Frau.
Verwende übrigens die schmale Platinen - Variante. Mit ein bischen Geschick beim Schrauben klappt das auch mit der AZ-Platinen Variante.

Noch ein Hinweis: Gibt es eventuell noch eine Halterung für den Stift vom Touchscreen? 
Da ich noch nicht bei FreeCad vorangekommen bin, kann ich das noch nicht selbst machen.

Danke schon mal im Voraus.

**Reply by paulchen67** [reply]
Auflage Verstärker ist nur zum abstützen des Verstärker zur Platine.
